test/controllers/PublicControllerSpec.scala

```diff
@@ -78,7 +78,7 @@ class PublicControllerSpec extends BaseSpec  {
 
       val r = c.redirectToExitSurvey(Origin("PERTAX"))(buildFakeRequestWithAuth("GET"))
       status(r) shouldBe SEE_OTHER
-      redirectLocation(r) shouldBe Some("/feedback-survey?origin=PERTAX")
+      redirectLocation(r) shouldBe Some("http://localhost:9514/feedback-survey?origin=PERTAX")
```

Comment:
Prod doesn't have a external-urls section, therefore the exit survey location is blank when it attempts to be read.

On dev, this is needed so it goes to the correct 9514 port. I assume this is the same for the processTaxCreditsChoice logic.